docs/ii-spec.mdx

```diff
@@ -393,15 +436,42 @@ The method returns the expiration timestamp of the delegation. This is returned
 
 The actual delegation can be fetched using `get_delegation` immediately afterwards.
 
+#### The `get_delegation` query method
 **Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
 
-### The `get_delegation` query method
-
 For a certain amount of time after a call to `prepare_delegation`, a query call to `get_delegation` with the same arguments, plus the timestamp returned from `prepare_delegation`, actually fetches the delegation.
 
 Together with the `UserKey` returned by `prepare_delegation`, the result of this method is used by the Frontend to pass to the client application as per the [client authentication protocol](#client-authentication-protocol).
 
-**Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
+### Verifiable Credentials Protocol
+
+See here for the specification of the verifiable credentials protocol.
```

Comment:
@jessiemongeon1: I'd like to link to the `vc-spec.md` file here. How do I do this correctly?

Reply:
```
See [here](vc-spec.md) for the specification of the verifiable credentials protocol.
```